subsys/net/lib/shell/sockets.c

```diff
@@ -123,10 +123,7 @@ static void walk_socket_services(const struct net_socket_service_desc *svc,
 	snprintk(owner, sizeof(owner), "<unknown>");
 #endif
 
-	PR("%32s  %-6s  %-5d %s\n",
```

Comment:
You forgot to change the commit subject of the shell changes.

Reply:
Oops